### PR TITLE
feat(Toolbar): add insets property

### LINKED ISSUE #4661: Toolbar: support for insets
https://github.com/patternfly/patternfly/pull/3317

This is a placeholder until the core work is done. This feature will be similar to the tabs component where insets that map to our spacer system can be used to define the outer padding of the toolbar component.

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Closes #4661

### CODE REVIEW

File: packages/react-integration/cypress/integration/toolbar.spec.ts [hunk 1]
```diff
@@ -5,6 +5,13 @@ describe('Data Toolbar Demo Test', () => {
     cy.url().should('eq', 'http://localhost:3000/toolbar-demo-nav-link');
   });
 
+  it('has inset classes', () => {
```
Comment (tlabaj) [inline packages/react-integration/cypress/integration/toolbar.spec.ts:8]: can you test all breakpoints.  We have had issues in the past when classes where not being applied correctly at certain breakpoints.  See Tabs for an example.

**Review by tlabaj — CHANGES_REQUESTED** [review]

**Review by tlabaj — APPROVED** [review]
LGTM

**Comment by mcarrano** [comment]
@mcoker @mattnolting can either of you confirm that this is doing what's it's supposed to?

**Review by evwilkin — APPROVED** [review]
LGTM

**Pull request merged**